File: src/main/java/me/nepnep/nepbot/message/command/commands/admin/RemoveLewdBlackList.java
package me.nepnep.nepbot.message.command.commands.admin;

import me.nepnep.nepbot.json.LewdBlackList;
import me.nepnep.nepbot.message.command.ICommand;
import me.nepnep.nepbot.message.command.Category;
import net.dv8tion.jda.api.Permission;
import net.dv8tion.jda.api.events.message.guild.GuildMessageReceivedEvent;

import java.util.List;

public class RemoveLewdBlackList implements ICommand {
    @Override
    public void execute(List<String> args, GuildMessageReceivedEvent event) {
        LewdBlackList.remove(event.getChannel().getIdLong());
    }
    @Override
    public String getInvoke() {
        return "removelewdblacklist";
    }
    @Override
    public Permission requiredPermission() {
        return Permission.MANAGE_CHANNEL;
    }
    @Override
    public Category getCategory() {
        return Category.ADMIN;
    }
    @Override
    public String getDescription() {
        return "Removes the current channel from the lewd blacklist";
    }
}

File: src/main/kotlin/me/nepnep/nepbot/message/command/commands/admin/RemoveLewdBlacklist.kt
package me.nepnep.nepbot.message.command.commands.admin

import me.nepnep.nepbot.database.BlacklistType
import me.nepnep.nepbot.database.removeFromBlacklist
import me.nepnep.nepbot.message.command.Category
import me.nepnep.nepbot.message.command.AbstractCommand
import net.dv8tion.jda.api.Permission
import net.dv8tion.jda.api.entities.GuildMessageChannel
import net.dv8tion.jda.api.events.message.MessageReceivedEvent

class RemoveLewdBlacklist : AbstractCommand(
    "removelewdblacklist",
    Category.ADMIN,
    "Removes the current channel from the lewd blacklist",
    Permission.MANAGE_CHANNEL
) {
    override fun execute(args: List<String>, event: MessageReceivedEvent, channel: GuildMessageChannel) {
        channel.removeFromBlacklist(BlacklistType.LEWD)
    }
}